From 7057efeb94d511cfaaee4d36e613e9f51d670911 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Tue, 16 Nov 2021 14:36:44 +0200
Subject: [PATCH] [SW-56241] boot_info: temporarily rename

Temporarily rename boot_info and related defines,
as a preparation for moving the struct to specs.

Change-Id: I291f111bf099d13d2ce4c573a767643f3a7ef042
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/lib/preboot/boot_info.c b/lib/preboot/boot_info.c
index 1e3f345..ab1c396 100644
--- a/lib/preboot/boot_info.c
+++ b/lib/preboot/boot_info.c
@@ -15,22 +15,22 @@
 #include "boot_info.h"
 #include "uart.h"
 
-#define MAX_INFO_DESCS	32
+#define _MAX_INFO_DESCS	32
 
-#define BOOT_INFO_SZ	(sizeof(struct hl_boot_info_ptrs) + \
-			sizeof(struct hl_boot_info_desc) * MAX_INFO_DESCS)
+#define _BOOT_INFO_SZ	(sizeof(struct hl_boot_info_ptrs) + \
+			sizeof(struct hl_boot_info_desc) * _MAX_INFO_DESCS)
 
-struct boot_info {
+struct _boot_info {
 	union {
 		struct {
 			struct hl_boot_info_ptrs ptrs;
-			struct hl_boot_info_desc descs[MAX_INFO_DESCS];
+			struct hl_boot_info_desc descs[_MAX_INFO_DESCS];
 		};
-		u32 data[ROUND_UP4(BOOT_INFO_SZ) / sizeof(u32)];
+		u32 data[ROUND_UP4(_BOOT_INFO_SZ) / sizeof(u32)];
 	};
 };
 
-static struct boot_info boot_info = { .ptrs = { .count = 0 } };
+static struct _boot_info boot_info = { .ptrs = { .count = 0 } };
 
 static inline void *offset2ptr(u32 offset, uintptr_t base)
 {
@@ -88,12 +88,12 @@
 void boot_info_finalize(void)
 {
 	struct hl_boot_info_desc *desc;
-	u32 offsets[MAX_INFO_DESCS];
+	u32 offsets[_MAX_INFO_DESCS];
 	u32 offset, copy_sz;
 	const u32 *src;
 	u32 *dst;
 	int i, j;
-	struct hl_boot_info_desc descs[MAX_INFO_DESCS];
+	struct hl_boot_info_desc descs[_MAX_INFO_DESCS];
 
 	if (!boot_info.ptrs.count)
 		return;
@@ -157,7 +157,7 @@
 
 	desc = boot_info_get_desc(type);
 	if (!desc) {
-		if (boot_info.ptrs.count >= MAX_INFO_DESCS) {
+		if (boot_info.ptrs.count >= _MAX_INFO_DESCS) {
 			preboot_puts_hex("reached max desc count ", type, 0);
 			return 1;
 		}
